File: Python3/URI1045.py
x = input().split()
A = float(x[0])
B = float(x[1])
C = float(x[2])
aux = 0
if C > B:
    aux = B
    B = C
    C = aux
if C > A:
    aux = A
    A = C
    C = aux
if B > A:
    aux = A
    A = B
    B = aux

if A >= B+C:
    print("NAO FORMA TRIANGULO")
else:
    if A ** 2 == B ** 2 + C ** 2:
        print("TRIANGULO RETANGULO")
    elif A ** 2 > B ** 2 + C ** 2:
        print("TRIANGULO OBTUSANGULO")
    elif A ** 2 < B ** 2 + C ** 2:
        print("TRIANGULO ACUTANGULO")
    if A == B and B == C:
        print("TRIANGULO EQUILATERO")
    elif A == B or A == C or B == C:
        print("TRIANGULO ISOSCELES")

File: C/URI1045.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main() {
    double a, b, c, aux;
    scanf("%lf %lf %lf", &a, &b, &c);
    if (c > b) {
        aux = b;
        b = c;
        c = aux;
    }
    if (c > a) {
        aux = a;
        a = c;
        c = aux;
    }
    if (b > a) {
        aux = a;
        a = b;
        b = aux;
    }
    
    if (a >= b + c)
        printf("NAO FORMA TRIANGULO\n");
    else {
        if (a * a  == b * b + c * c)
        printf("TRIANGULO RETANGULO\n");
        if (a * a  > b * b + c * c)
            printf("TRIANGULO OBTUSANGULO\n");
        if (a * a  < b * b + c * c)
            printf("TRIANGULO ACUTANGULO\n");
        if (a == b && b == c)
            printf("TRIANGULO EQUILATERO\n");
        else if (a == b || b == c)
            printf("TRIANGULO ISOSCELES\n");
    }

    return 0;

}